crates/bevy_ui/src/render/mod.rs

```diff
@@ -264,7 +264,7 @@ pub fn extract_uinode_borders(
                         stack_index: node.stack_index,
                         // This translates the uinode's transform to the center of the current border rectangle
                         transform: transform * Mat4::from_translation(edge.center().extend(0.)),
-                        color: border_color.0,
+                        color: border_color.0.into(),
```

Comment:
I was wondering if sometime in the future we can implement deref and derefmut for this types that are only wrappers in ui, would simplify some places

Reply:
Yeah, I'm not opposed to that.